CodeMeKathy/glowing-memory
# Feature: Install, connect and setup firebase with database and hosting

## Description


This Pull Request (PR) features installing the Firebase dependency and connects and setups the Firebase database along with Firebase hosting for the app at https://shoppinglistapp-783fc.web.app.

## Related Issue


Closes: #2

## Acceptance Criteria


- [X] The following have been added as project dependencies: firebase.

## Type of Changes


|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✅   | :sparkles: New feature     |
| ✅     | :hammer: Refactoring       |
|     | :100: Add tests            |
| ✅   | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before


<details markdown=block>
<summary markdown=span>Initital Shopping List UI Layout</summary>

- Shopping List (`<List/>`):
![`<List/>_Before_Empty_List`](https://user-images.githubusercontent.com/32943879/228951709-176346c1-2287-4e23-9ff2-0186b2611c88.png)

</details>

### After


<details markdown=block>
<summary markdown=span>Updates</summary>

- Shopping List `<List/>_Empty_List`):
![`<List/>_After_Empty_List`](https://user-images.githubusercontent.com/32943879/228951102-6fce0785-3a73-4370-ade5-f09de0c81f51.png)


</details>

## Testing Steps / QA Criteria
 

- From your terminal, run `kl-firebase-connection`.
- Once on the `kl-firebase-connection` branch in the browser, visit the new `<Home/>` component (http://localhost:3000/list).
- Access the Web Developer Tools to view the Console Log Firebase Snapshot information (see enclosed image) with database "projectId" confirming Firebase connectivity.
